source/web/HashingTools.js

```diff
@@ -0,0 +1,105 @@
+"use strict";
```

Comment:
Do we need use strict? I believe this file is a bit old

Reply:
Nope! We do not. I'll remove it.